### PR TITLE
Create SieveofEratosthenes.py

### PR DESCRIPTION
**Pull request by swap1010**
implemented sieve of Eratosthenes in one line

### CODE REVIEW

**Review by girishtulabandu — CHANGES_REQUESTED** [review]
Kindly add the comment in the first line which explains your code

**Comment by swap1010** [comment]
Sure

File: SieveofEratosthenes.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// The one-liner uses the reduce function to remove, one step at a time,
+//all “marked” numbers from the initial set of all numbers between 2 and n (in the one-liner: set(range(2, n))).
```
Comment (girishtulabandu) [inline SieveofEratosthenes.py]: Kindly comment properly. In python, Use # for single line comment and triple quotes (" " ") for multiline comments

Comment (swap1010) [reply SieveofEratosthenes.py]: done

**Comment by swap1010** [comment]
Done sir

**Pull request merged**